behat.yml

```
@@ -156,6 +157,7 @@ legacy:
                 selenium2:
                     selenium2:
                         wd_host: 'http://selenium:4444/wd/hub'
+                        capabilities: { "browserName": "chrome", "browser": "chrome", 'chrome': {'switches':['--no-sandbox']}}
```

```suggestion
                        capabilities: { "browserName": "chrome", "browser": "chrome", "chrome": {"switches":["--no-sandbox"]}}
```

JSON-like values with single quotes annoys me :smile: